pkg/cross-compilers/build.yml

```diff
@@ -0,0 +1,3 @@
+image: eve-cross-compilers
+org: lfedge
+network: yes
```

Comment:
Not only. Aport scripts also use network to download sources of pacakges.

Reply:
@giggsoff from a build reproducibility and SBOM perspective we want to avoid network: yes and instead rely on ADD commands. Is that possible for aports?

Reply:
I put required source archives into Dockerfile, let's see how it will pass CI.